sv:
  use_transform: false
  voxel_resolution: 0.008
  color_importance: 0.2
  spatial_importance: 0.4
  normal_importance: 0.4
  seed_resolution: 0.05
dataset:
  int numClasses: 2
  numFeatures: 6
  minColorRange: 0.0
  maxColorRange: 255.0
  minNormalRange: -1.0
  maxNormalRange: 1.0
workspace:
  sphere:
    x: 0.
    y: 0.
    z: 1.
    radius: 10.
    threshold: 0.35
  csg_intersect_cuboid:
    x_min: -0.38
    x_max: 0.2
    y_min: -0.1
    y_max: 0.35
    z_min: .85
    z_max: 2.0
camera_parameters:
  rgb:
    focal_length:
      x: 570.342
      y: 570.342
    principal_point:
      x: 319.5
      y: 239.5
  depth:
    focal_length:
      x: 570.342
      y: 570.342
    principal_point:
      x: 314.5
      y: 235.5
camera_to_robot_frame_transformation:
  translation :
    x: 1.21
    y: 0.2
    z: 0.33
  rotation :
    x: -0.438
    y: -0.031
    z: 0.898
    w: -0.016
